std_msgs/Header header

string note

# plotjuggler with stick plots need a constant data item to show 
float32 confidence
